### DISCUSSION TITLE
How v-slot property work with vue3&&tsx？

### DISCUSSION THREAD

**Discussion by gideonsenku** [tailwindlabs/headlessui#373] [Help]
### In my project, i use vue3 and tsx, but there has a problem when i use MenuItem,just look like this:
<img width="961" alt="截屏2021-04-16 上午10 12 56" src="https://user-images.githubusercontent.com/39037656/114961907-53073480-9e9c-11eb-9bb0-dd31ad83f0a7.png">

**Comment by RobinMalfait** [comment]
Hey! Thank you for your bug report!
Much appreciated! 🙏

Can you create a minimal reproduction repo and share it?

**Reply by gideonsenku** [reply]
## I have been resolve this problem that destructuring slot props,it's not a bug, thank u.
## And there is my resolved:
<img width="914" alt="截屏2021-04-17 下午2 56 10" src="https://user-images.githubusercontent.com/39037656/115104750-0e0efb00-9f8d-11eb-8ad5-adc59597cca5.png">
